### DISCUSSION TITLE
Show images only in output buffer and not in virtual text

### DISCUSSION THREAD

**Discussion by michaelbrusegard** [benlubas/molten-nvim#236] [Q&A]
I am using WezTerm and the images in virtual text is quite buggy. But when they show up in the output buffer they are always rendered correctly. I would like to keep virtual text enabled since it is quite handy for non-image output. Is there a way I could keep virtual text output while having images only render when I open the output buffer?

**Comment by benlubas** [comment]
Good idea, its not currently possible, you could open a feature request issue and I'll see when I have time for it

**Reply by michaelbrusegard** [reply]
I will, do you know why it is more buggy when the image is outputted into virtual text? Is it because I am using WezTerm. (Note: It works perfectly in the output buffer)

**Reply by michaelbrusegard** [reply]
https://github.com/benlubas/molten-nvim/issues/237

**Reply by benlubas** [reply]
Wezterm does have less than ideal image rendering. But there was a small update to image.nvim a while ago that changed the position of images relative to their ext mark that I haven't adapted to in molten yet. So it looks a little bit weird right now.
